routes/books.js

```diff
@@ -27,16 +27,17 @@ router.route('/')
     ])
   })
   .post(function(req, res) {
-    res.statusCode = 200;
-    res.json({
-      "id": 127,
-      "user_id": 1,
-      "title": "Test Book 3: The Bookening",
-      "author": "An Author Name",
-      "isbn": "1234567890123",
-      "summary": "An Author Name concludes his thrilling horror series with a new age children's book",
-      "visibility": true
-    })
+    let book = req.body;
+    db.books.insert(book).then((result) => {
```

Comment:
Instead of using `.then((result) =>`, I would use `.then(([result, fields]) =>`
This will allow you to use `result.affectedRows` instead of `result[0].affectedRows`